contrib/docker/Dockerfile

```diff
@@ -1,5 +1,10 @@
-FROM docker.io/library/golang:1.18 as builder
+# TARGETPLATFORM and BUILDPLATFORM are automatically field when specifying an architecture during build but they need to be declared
```

Comment:
field -> filled

Reply:
Thanks, I changed a bit this comment to be more accurate with the last commits.